engine/install/ubuntu.md

```diff
@@ -42,7 +42,15 @@ $ sudo apt-get remove docker docker-engine docker.io containerd runc
 It's OK if `apt-get` reports that none of these packages are installed.
 
 The contents of `/var/lib/docker/`, including images, containers, volumes, and
-networks, are preserved. The Docker Engine package is now called `docker-ce`.
+networks, are preserved. If you do not need to save anything or you want to
+start with a clean installation, it is safe to remove this directory with the
+command:
+
+```bash
+$ sudo rm -rf /var/lib/docker
+```
```

Comment:
So, while I think it's correct, I'm a bit concerned about adding  this example in this section. This section is for situations where people are upgrading from either very old versions of docker, or if they previously had a distro-packaged version of docker installed.

My concern is that, while users _should_ read carefully, it's also easy to overlook why this line is here, and blindly copy/paste it to their terminal and end up with all (possibly important) data gone.

Perhaps instead refer them to the "uninstall docker" section if they want to do a clean install; https://docs.docker.com/engine/install/ubuntu/#uninstall-docker-engine

```suggestion
networks, are preserved. If you do not need to save your existing data, and want to
start with a clean installation, refer to the [uninstall Docker Engine](#uninstall-docker-engine)
section at the bottom of this page.
```

WDYT?

Reply:
I agree, it's potentially dangerous.

Your suggested edit is good.